wait/exec.go

```diff
@@ -72,13 +82,16 @@ func (ws *ExecStrategy) WaitUntilReady(ctx context.Context, target StrategyTarge
 		case <-ctx.Done():
 			return ctx.Err()
 		case <-time.After(ws.PollInterval):
-			exitCode, _, err := target.Exec(ctx, ws.cmd)
+			exitCode, resp, err := target.Exec(ctx, ws.cmd, tcexec.Multiplexed())
```

Comment:
We used multiplexed to remove the trim the trailing bytes from the Docker protocol (garbage). See #624